app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/ui/output/OutputPresenter.java

```diff
@@ -3,12 +3,11 @@
 import com.benoitquenaudon.daggerbundleinjectiondemo.injection.SomeInput;
 import javax.inject.Inject;
 
-public class OutputPresenter {
+class OutputPresenter {
   @Inject @SomeInput String inputText;
-  private final OutputActivity activity;
+  @Inject OutputActivity activity;
```

Comment:
I usually always inject everything in the constructor when I can. Because it allows you to pass manually whatever you want when testing it and I think it keeps the code clean.
I put `@inject` on fields only when there is no other choice (like Activity).

Although I wrote `@Inject @SomeInput String inputText`, I'd put it in the constructor if I reviewed myself properly :D